scripts/qa_crowbarsetup.sh

```diff
@@ -799,8 +799,10 @@ BROADCAST='$net.255'
 EOF
     ifdown br0
     rm -f /etc/sysconfig/network/ifcfg-br0
-    grep -q "^default" /etc/sysconfig/network/routes || \
```

Comment:
Maybe the grep was there to leave different default route intact.
How about adding a 2>/dev/null instead of another variable and test?